.travis.yml

```diff
@@ -9,13 +9,18 @@ osx_image: xcode6.4
 env:
   matrix:
     
-    - CONDA_R-BASE=3.3.2
+    - CONDA_R=3.3.2
```

Comment:
`CONDA_R` should be the one to use (ensuring consistency with Continuum and Bioconda). It also makes more sense when comparing to how the pinning in other languages is named (CONDA_PY, CONDA_PERL, ...).